### PR TITLE
x11-apps/copyq: Bump to Version 3.7.3

### PR DESCRIPTION
**Pull request by oniGino**
* Version Bump
* Drop old release

### CODE REVIEW

File: x11-apps/copyq/copyq-3.7.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline x11-apps/copyq/copyq-3.7.3.ebuild]: bump to 7 :)

Comment (oniGino) [reply x11-apps/copyq/copyq-3.7.3.ebuild]: ok bumped

Comment (oniGino) [reply x11-apps/copyq/copyq-3.7.3.ebuild]: @Zlogene please review again

**Comment by oniGino** [comment]
@Zlogene please review

**Comment by oniGino** [comment]
@SoapGentoo filed upstream https://github.com/hluk/CopyQ/issues/1123

**Pull request closed without merging**